tests/integration/helpers.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2021 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import asyncio
+import json
+import logging
+import urllib.request
+from pathlib import Path
+from typing import List
+
+log = logging.getLogger(__name__)
+
+
+async def cli_deploy_bundle(ops_test, name: str, channel: str = "edge"):
```

Comment:
Why are we using this rather than `ops_test.build_charm`? Is it because certain options aren't exposed? If so, there should be a comment about what option make this necessary and we should open a bug with pytest-operator.

Edit: I see the comment in another file about the bug. Maybe just duplicate that comment here.

Reply:
Added.